#!/usr/bin/env python
PACKAGE = 'tams_pr2_guzheng'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("active", bool_t, 1<<0, "active", True)
gen.add("reject_short_strings", bool_t, 1<<1, "reject short strings", True)
gen.add("reject_unexpected", bool_t, 1<<2, "reject unexpected", False)
gen.add("reject_unaligned", bool_t, 1<<3, "reject unaligned", False)
gen.add("align_heads", bool_t, 1<<4, "align heads", False)

exit(gen.generate(PACKAGE, "", "StringFitter"))
